xml/deployment_prep_power.xml

```diff
@@ -25,16 +12,522 @@ fate#304108: PPC64 Hotplug memory remove
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry>
+         <para>
+          All patterns
+         </para>
+        </entry>
+        <entry>
+         <para>
+          4 GB
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry namest="col1" nameend="col2">
+         <para>
+          Recommended Minimum (no Btrfs snapshots): 10 GB
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry namest="col1" nameend="col2">
+         <para>
+          Required Minimum (with Btrfs snapshots): 16 GB
+         </para>
+        </entry>
+       </row>
+       <row>
+        <entry namest="col1" nameend="col2">
+         <para>
+          Recommended Minimum (with Btrfs snapshots): 32 GB
+         </para>
+        </entry>
+       </row>
+      </tbody>
+     </tgroup>
+    </informaltable>
+    <para>
+     If your root partition is smaller than 10 GB, the installer will not make
+     an automated partitioning proposal and you need to manually create
+     partitions. Therefore the recommended minimum size for the root partition
+     is 10 GB. If you want to enable Btrfs snapshots on the root volume to
+     enable system rollbacks (see <xref linkend="cha-snapper"/>) the minimum
+     size for the root partition is 16 GB.
+    </para>
+   </listitem>
+  </varlistentry>
+ </variablelist>
+ <para>
+  Your servers should be equipped with the latest firmware before installing
+  &productname;. Find up-to-date firmware at IBM FixCentral (<link
+  xlink:href="http://www.ibm.com/support/fixcentral/"/>). Select your system
+  from the Product Group list. Additional software is available from the IBM
+  &powerlinux; Tools Repository. The IBM Tools Repository is also called the
+  Yum Repository. For more information about using the IBM &powerlinux; Tools
+  Repository, see <link xlink:href="https://ibm.biz/Bdxn3N"/>.
+ </para>
+</sect1>
+
+<sect1 version="5.0" xml:id="sec-power-install"
+       xmlns="http://docbook.org/ns/docbook"
+       xmlns:xi="http://www.w3.org/2001/XInclude"
+       xmlns:xlink="http://www.w3.org/1999/xlink">
+  <title>Installing &sls; for &power;</title>
+  <para>
+    The following procedure describes how to set up an installation environment.
+    You can skip it if you already have an installation enviroment ready.
+  </para>
+  <procedure>
+    <title>Preparing an Installation Environment</title>
+    <step>
+      <para>
+	Start an SSH session to your HMC and run the <command>vtmenu</command> command.
+      </para>
+    </step>
+    <step>
+      <para>
+	Select the desired &power; server and the LPAR. If a serial console session for the
+	chosen LPAR already exists, you need to close it first using the following command:
+      </para>
+      <screen>rmvterm -m <replaceable>SERVER</replaceable> -p <replaceable>LPAR</replaceable></screen>
+      <para>
+	Note that this command causes a hard reboot of the LPAR. To perform a soft reboot
```

Comment:
There is some text missing here. The `rmvterm` command does not reboot the LPAR. The command which does is:

`chsysstate -r lpar -m <server-name> -o shutdown -n <lpar-name> --immed --restart`

Reply:
Oops! Fixed.